Bitrise/tag_releasing_bitrise.yml

```diff
@@ -44,6 +44,15 @@ workflows:
             # Use the environment FASTLANE_TAG_RELEASE_LANE if available. Otherwise, fallback to "release_from_tag"
             lane=${FASTLANE_TAG_RELEASE_LANE:=release_from_tag}
             fastlane $lane
+    - script:
+        title: Clean up after failure
+        is_always_run: true
+        run_if: .IsBuildFailed
```

Comment:
This might not be the correct check according to [this](https://devcenter.bitrise.io/en/builds/build-statuses.html#build-statuses), but I couldn't find anywhere that mentions having a `.IsBuildAborted`.

Reply:
I'll go with it:
```suggestion
        run_if: .IsBuildAborted
```

Reply:
> In general we consider aborted builds as failures, because otherwise it could be misleading / you might miss an important issue.

As read from [here](https://discuss.bitrise.io/t/how-to-programmatically-abort-instead-of-fail-a-build/1791) so I guess we're good!

Reply:
I reverted the `.IsBuildAborted`. Thanks for the tip! I've found your quote [here](https://discuss.bitrise.io/t/how-to-programmatically-abort-instead-of-fail-a-build/1791).